src/aura/BGE_TemplateDetails/BGE_TemplateDetailsController.js

```diff
@@ -0,0 +1,139 @@
+({
+
+    /******************************** Template Details Controller Functions *****************************/
+
+    /* ***************************************************************
```

Comment:
I prefer the smaller comments in JS and will change these to match. The header comment is helpful because of the module pattern used here, and would be helpful to keep.